src/AppInstallerCommonCore/Public/winget/ManifestInstaller.h

```diff
@@ -71,6 +71,6 @@ namespace AppInstaller::Manifest
         // For msix only
         std::vector<string_t> RestrictedCapabilities;
 
-        Dependency Dependencies;
+        DependencyList Dependencies;

```

Comment:
Are these properly being applied to all installers if they are in the root of the manifest?  I don't remember if the manifest schema allows that; but if it does there is extra work to ensure that it actually happens.

Reply:
@denelon said the common behavior is: 
- Root applies when none are specified on an installer node. 
- Installer nodes override root. 

Added tests for this, change it to work like that and put a comment on this features' specs.